tests/test_deadlock.py

```diff
@@ -0,0 +1,51 @@
+import uuid
+
+import coiled.v2
+import dask
+from distributed import Client
+
+
+def test_deadlock(software):
```

Comment:
I'd rather we aimed towards correctness, and called this "test shuffle" or something similar.  When things are working this should test that a certain user-focused workload is solid.  I think that that should influence the name.

Reply:
Sure, this was just a temporary filler name